libsolidity/ast/AST.h

```diff
@@ -152,6 +152,9 @@ class Declaration: public ASTNode
 	/// Visibility ordered from restricted to unrestricted.
 	enum class Visibility { Default, Private, Internal, Public, External };
 
+	// How the function can mutate the EVM state.
+	enum class StateMutability { Pure, View, NonPayable, Payable };
```

Comment:
Placing it here will not work due to forward declarations hiding it. Any suggestions for where to place this?